src/modules/CanvasResize/Module.js

```diff
@@ -1,6 +1,8 @@
+const ndarray = require('ndarray');
 /*
 * Changes the Canvas Size
 */
+const getPixels = require('get-pixels');
```

Comment:
This new import is added but doesnt look like it is used anywhere.

Reply:
```suggestion
```